[DO NOT MERGE] Update TFC Documentation for Cloud Blocks

laurapacilio opened the pull request:
I have:
- [x] Built this locally.
- [x] Added at least one label.
- [x] Added all required reviewers. (please feel free to comment even if you aren't on the list, though!)

----------------

**DESCRIPTION**

This PR updates the following pages in the Terraform Cloud documentation for the v1.1 release:

**[Workspaces Index](https://www.terraform.io/docs/cloud/workspaces/index.html)** 
- Remove "see" in references to other pages (avoiding ableist language, per style guide)
- Update the Workspaces Terraform Cloud UI element (to bold, not quotes) to adhere to the new style guide guidance. 
- Create a new subsection that clarifies how TFC workspaces differ from CLI workspaces and points to the Language workspaces page for more on CLI workspaces. The short explanations are adapted from text in the companion /terraform PR for cloud documentation and also on a note that was previously in the text warning users that CLI and TFC workspaces are different. Given our discussions about how this is a pain point, I wanted to try to do something that would move us in the right direction and (hopefully) provide more explicit clarity without being too heavy of a lift. 
- Move the learn callout and the note about the API and move them to a new section called "Creating Workspaces". The Learn callout felt like it came too soon at the very top of the page before users had even had a chance to learn about what workspaces are. The pointer to the entire get started collection also felt too broad, and I thought users might appreciate a more specific pointer, so I updated it to point to the "Create a Workspace" tutorial in the collection. The API note also felt a bit abrupt in that section, so I tried grouping it in the "Create a Workspace" section with other relevant pointers :-) 

**[Terraform State](https://www.terraform.io/docs/cloud/workspaces/state.html)**
- Remove mention of remote backend and change to new cloud block. 
- Add note about how the cloud block is only available in Terraform v1.1 and TFE v202201..

**[The UI and VCS Driven Workflow](https://www.terraform.io/docs/cloud/run/ui.html)**
Remove mention of remote backend from end of page.

**[The CLI-driven Run Workflow](https://www.terraform.io/docs/cloud/run/cli.html)**
- Change language so that document addresses the reader as "you", according to our style guide.
- Change references to remote backend to CLI integration and where appropriate `cloud` block. Link to corresponding documentation.
- Remove CLI output examples that reference the remote backend. I decided NOT to replace these because I don't know that they are adding much to the user's understanding. Removing them shortens the page. 
- Changed variables section to reflect the new variable sets and per-run variables features.
- Add note about how the cloud block is only available in Terraform v1.1 and TFE v202201.

**[The Run Environment](https://www.terraform.io/docs/cloud/run/run-environment.html)**
- Remove references to remote backend and update links to new cloud docs.
- Change "private module registry" to "private registry" in accordance with recent changes.
- Reorganize the State Access and Authentication section. Add subheadings to help break up the text/hopefully make it easier for users to parse. Do a few language edits to make sure it adheres to our style guide/flows better :-) 
- Add note about how the cloud block is only available in Terraform v1.1 and TFE v202201.

**[Migrating from Local Terraform](https://www.terraform.io/docs/cloud/migrate/index.html)**
- Rename to "Migrating to Terraform Cloud" 
- Add note about how the cloud block is only available in Terraform v1.1 and TFE v202201.
- Remove existing "how to" content and instead include pointers to places where users can learn how to migrate from both CLI and VCS. How-to CLI content will be merged into the CLI docs detailing how to use the `cloud` block.

**Additional Mentions**
I performed a search for "`remote`" and "remote backend" in the cloud docs folder and found the following files that had mentions of the remote backend. I remove/change those mentions in each file. Where possible, I refer to this as the "CLI integration" rather than using the name of the block. This protects us in case implementation changes again in the future.
- [API Docs Index](https://www.terraform.io/docs/cloud/api/index.html)
- [Workspaces API page](https://www.terraform.io/docs/cloud/api/workspaces.html)
- [Overview of Terraform Cloud Features](https://www.terraform.io/docs/cloud/overview.html)
- [Terraform Runs and Remote Operations](https://www.terraform.io/docs/cloud/run/index.html)
- [Terraform Configurations in Terraform Cloud Workspaces](https://www.terraform.io/docs/cloud/workspaces/configurations.html)
- [Workspace Settings](https://www.terraform.io/docs/cloud/workspaces/settings.html)

robin-norwood commented on content/source/docs/cloud/run/run-environment.html.md:
```diff
@@ -42,49 +42,46 @@ Terraform Cloud can also delay some runs in order to make performance more consi
+|`TFC_RUN_ID` | A unique identifier for this run (e.g. `"run-CKuwsxMGgMd4W7Ui"`). |
+|`TFC_WORKSPACE_NAME` | The name of the workspace used in this run, e.g. `"prod-load-balancers"`. |
+|`TFC_WORKSPACE_SLUG` | The full slug of the configuration used in this run. This consists of the organization name and workspace name, joined with a slash, e.g. `"acme-corp/prod-load-balancers"`. |
+|`TFC_CONFIGURATION_VERSION_GIT_BRANCH` | The name of the branch that the associated Terraform configuration version was ingressed from (e.g. `main`). |
```
```suggestion
|`TFC_CONFIGURATION_VERSION_GIT_BRANCH` | The name of the branch that the associated Terraform configuration version was ingressed from, e.g. `main`. |
```

robin-norwood commented on content/source/docs/cloud/run/run-environment.html.md:
```diff
@@ -42,49 +42,46 @@ Terraform Cloud can also delay some runs in order to make performance more consi
-  (e.g. `"v0.1.0"`).
+| Variable Name | Description |
+| --------------| ------------|
+|`TFC_RUN_ID` | A unique identifier for this run (e.g. `"run-CKuwsxMGgMd4W7Ui"`). |
```
Well, ok, it's 50/50 => what about adding an "Example" column instead of `e.g.`?

robin-norwood commented on content/source/docs/cloud/run/run-environment.html.md:
```diff
@@ -42,49 +42,46 @@ Terraform Cloud can also delay some runs in order to make performance more consi
+|`TFC_WORKSPACE_NAME` | The name of the workspace used in this run, e.g. `"prod-load-balancers"`. |
+|`TFC_WORKSPACE_SLUG` | The full slug of the configuration used in this run. This consists of the organization name and workspace name, joined with a slash, e.g. `"acme-corp/prod-load-balancers"`. |
+|`TFC_CONFIGURATION_VERSION_GIT_BRANCH` | The name of the branch that the associated Terraform configuration version was ingressed from (e.g. `main`). |
+| `TFC_CONFIGURATION_VERSION_GIT_COMMIT_SHA` | The full commit hash of the commit that the associated Terraform configuration version was ingressed from (e.g. `"abcd1234..."`). |
+|`TFC_CONFIGURATION_VERSION_GIT_TAG` | The name of the tag that the associated Terraform configuration version was ingressed from (e.g. `"v0.1.0"`). |
```
```suggestion
| `TFC_CONFIGURATION_VERSION_GIT_COMMIT_SHA` | The full commit hash of the commit that the associated Terraform configuration version was ingressed from, e.g. `"abcd1234..."`. |
|`TFC_CONFIGURATION_VERSION_GIT_TAG` | The name of the tag that the associated Terraform configuration version was ingressed from, e.g. `"v0.1.0"`. |
```

robin-norwood reviewed (approved):
LGTM! A couple of suggestions.

nfagerlund commented on content/source/docs/cloud/run/cli.html.md:
```diff
@@ -41,63 +41,50 @@ To supplement these remote operations, you can also use the optional [Terraform
 Automatically created workspaces might not be immediately ready to use, so use Terraform Cloud's UI to check a workspace's settings and data before performing any runs. In particular, note that:
 
-- No Terraform variables or environment variables are created by default. Terraform Cloud will use `*.auto.tfvars` files if they are present, but you will usually still need to set some workspace-specific variables.
+- No Terraform variables or environment variables are created by default, unless your organization has configured one or more [global variable sets](/docs/cloud/workspaces/variables.html#scope). Terraform Cloud will use `*.auto.tfvars` files if they are present, but you will usually still need to set some workspace-specific variables.
```
Ah good catch.

nfagerlund commented on content/source/docs/cloud/run/cli.html.md:
```diff
@@ -41,63 +41,50 @@ To supplement these remote operations, you can also use the optional [Terraform
 
-- Terraform variables and environment variables set in the workspace. These can be edited via the UI, the API, or the `tfe` Terraform provider.
-- Terraform variables from any `*.auto.tfvars` files included in the configuration. Workspace variables, if present, override these.
+- Run-Specific variables set via the command line or in your local environment. Terraform only uses the environment variables from your shell environment that are prefixed with `TF_VAR`.
+- Workspace-Specific Terraform and environment variables set in the workspace.
```
I wouldn't capitalize "Specific" here.

laurapacilio replied:
ah you're right! thank you :)

nfagerlund commented on content/source/docs/cloud/run/cli.html.md:
```diff
@@ -111,78 +98,31 @@ If you use a combined repository and [specify a working directory on workspaces]
 ## Remote Applies
 
-When configuration changes are ready to be applied, use the `terraform apply` command. The apply will start in Terraform Cloud, and the command line will prompt for approval before applying the changes.
+You can trigger remote applies in any workspaces where you have permission to apply runs. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```
Not quite true! 

```suggestion
In workspaces that aren't connected to a VCS repository, users with permission to apply runs can trigger remote applies with the CLI. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

The reasons for this are too complicated to get into right now, but that's how it currently is.

nfagerlund reviewed (approved):
This looks great. Thank you for taking this on!

judithpatudith commented on content/source/docs/cloud/migrate/index.html.md:
```diff
@@ -1,124 +1,32 @@
+## Requirements
 
-## Step 4: Prepare Your Terraform Working Directory
+Stop all Terraform runs before migrating state files. You should also only migrate state files into Terraform Cloud workspaces that have never performed a run.
```
I think this applies to the current workspace the state files apply to (as opposed to runs in your TFC org)?

laurapacilio replied:
yesss I think you're right. I'm going to specify - thank you!

laurapacilio replied:
Actually i'm confused - @korinne or @chrisarcand This was in the docs before I came in but did we mean "stop all runs locally"? The reason why is it doesn't make sense to tell users to do this in TFC if they have to migrate state into new workspaces that have never performed a run anyway... Unless the guidance actually IS to make it so that NO runs are happening in your TFC org (which seems disruptive).

im2nguyen replied:
I think this just means to stop all runs (local and/or remote) before you're migrating state files. This means that you should either wait for any running `apply` to finish, or to cancel them

laurapacilio replied:
okay I pushed up a change that I think helps clarify @im2nguyen will you take a look when you do your review? :-)

im2nguyen replied:
LGTM! Would like @korinne or @chrisarcand to confirm before you resolve this tho 😄

laurapacilio replied:
Chris has confirmed that this looks good and we made a tiny tweak to the language as well. I'll push up the change and then I think we should be good to go!! Thank you!

judithpatudith commented on content/source/docs/cloud/api/workspaces.html.md:
```diff
@@ -63,7 +63,7 @@ Key path                                      | Type    | Default   | Descriptio
 `data.attributes.source-name` **(beta)**      | string  | (nothing) | A friendly name for the application or client creating this workspace. If set, this will be displayed on the workspace as "Created via `<SOURCE NAME>`".
 `data.attributes.source-url` **(beta)**       | string  | (nothing) | A URL for the application or client creating this workspace. This can be the URL of a related resource in another app, or a link to documentation or other info about the client.
-`data.attributes.speculative-enabled`         | boolean | true      | Whether this workspace allows automatic [speculative plans][]. Setting this to `false` prevents Terraform Cloud from running plans on pull requests, which can improve security if the VCS repository is public or includes untrusted contributors. It doesn't prevent manual speculative plans via the remote backend or the runs API.
+`data.attributes.speculative-enabled`         | boolean | true      | Whether this workspace allows automatic [speculative plans][]. Setting this to `false` prevents Terraform Cloud from running plans on pull requests, which can improve security if the VCS repository is public or includes untrusted contributors. It doesn't prevent manual speculative plans via the remote CLI workflow or the runs A ...
```
```suggestion
`data.attributes.speculative-enabled`         | boolean | true      | Whether this workspace allows automatic [speculative plans][]. Setting this to `false` prevents Terraform Cloud from running plans on pull requests, which can improve security if the VCS repository is public or includes untrusted contributors. It doesn't prevent manual speculative plans via the CLI workflow or the runs API.
```

judithpatudith commented on content/source/docs/cloud/workspaces/configurations.html.md:
```diff
@@ -32,7 +31,7 @@ There are two ways to provide configuration versions for a workspace:
 
 - **With direct uploads.** You can use a variety of tools to directly upload configuration content to Terraform Cloud:
-    - **Terraform CLI:** With [the `remote` backend][remote backend] configured, the `terraform plan` and `terraform apply` commands will perform remote runs by uploading a configuration from a local working directory. See [The CLI-driven Run Workflow](../run/cli.html) for more information.
+    - **Terraform CLI:** With [CLI integration][/docs/cli/configuring-terraform-cloud/index.html] configured, the `terraform plan` and `terraform apply` commands will perform remote runs by uploading a configuration from a local working directory. See [The CLI-driven Run Workflow](../run/cli.html) for more information.
```
```suggestion
    - **Terraform CLI:** With the [CLI integration][/docs/cli/configuring-terraform-cloud/index.html] configured, the `terraform plan` and `terraform apply` commands will perform remote runs by uploading a configuration from a local working directory. See [The CLI-driven Run Workflow](../run/cli.html) for more information.
```

laurapacilio replied:
fixed!

judithpatudith reviewed (approved):
Great Job! Just a few tiny nits but I don't have anything big. Going to pre-approve 🎉 
- The listed version of enterprise won't be out for a while. Are we comfortable making future-facing statements in the docs about when the cloud block will be available? 
- In general I'd prefer keeping the code blocks and updating the output. It's nice to confirm that things are working like users would expect, and can also help with our SEO if people google a particular output. 
- Version notes have inconsistent formatting across pages

laurapacilio commented on content/source/docs/cloud/migrate/index.html.md:
```diff
@@ -1,124 +1,32 @@
-- [Set the VCS repository](../workspaces/vcs.html#vcs-connection-and-repository). This is optional, but enables automatic Terraform runs on code changes and automatic plans on pull requests. For more information, see [The UI- and VCS-driven Run Workflow](../run/ui.html).
-- [Set values for variables](../workspaces/variables.html).
-- [Set team access permissions](../workspaces/access.html).
+To migrate via the Terraform CLI, you can add the `cloud` block to your configuration, specify one or more Terraform Cloud workspaces for the state files, and run `terraform init`. If the workspaces you choose do not yet exist, Terraform Cloud creates them automatically in the specified organization.
```
Added locally and will push up :-)

laurapacilio commented on content/source/docs/cloud/run/cli.html.md:
```diff
@@ -23,13 +23,13 @@ Terraform Cloud has three workflows for managing Terraform runs.
 ## Summary
 
-The [Terraform remote backend][remote] brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It offers the best of both worlds to developers who are already comfortable with using Terraform, and can work with existing CI/CD pipelines.
+The [CLI integration][cloud] brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It offers the best of both worlds to developers who are already comfortable with using Terraform, and it can work with existing CI/CD pipelines.
```
gotcha - changed locally and will push up!

laurapacilio commented on content/source/docs/cloud/run/index.html.md:
```diff
@@ -61,7 +61,7 @@ Terraform Cloud has three main workflows for managing runs, and your chosen work
 - The [CLI-driven run workflow](./cli.html), which uses Terraform's standard CLI tools to execute runs in Terraform Cloud.
 
-In more abstract terms, runs can be initiated by VCS webhooks, the manual "Start new run" action in the workspace actions menu, the standard `terraform apply` command (with the remote backend configured), and [the Runs API](../api/run.html) (or any tool that uses that API).
+In more abstract terms, runs can be initiated by VCS webhooks, the manual "Start new run" action in the workspace actions menu, the standard `terraform apply` command (with the CLI integration configured), and [the Runs API](../api/run.html) (or any tool that uses that API).
```
you're right - thank you so much - fixed locally and will push up!

laurapacilio commented on content/source/docs/cloud/workspaces/index.html.md:
```diff
@@ -7,8 +7,6 @@ description: |-
 
 # Workspaces
 
-> **Hands-on:** Try the [Get Started - Terraform Cloud](https://learn.hashicorp.com/collections/terraform/cloud-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn.
-
 Workspaces are how Terraform Cloud organizes infrastructure.
```
good call - I'm going to remove this top bit - we don't need it :) And the rest is intro content.

laurapacilio commented on content/source/docs/cloud/workspaces/index.html.md:
```diff
@@ -19,9 +17,6 @@ When run locally, Terraform manages each collection of infrastructure with a per
 
 Terraform Cloud manages infrastructure collections with _workspaces_ instead of directories. A workspace contains everything Terraform needs to manage a given collection of infrastructure, and separate workspaces function like completely separate working directories.
 
```
that's a good point - Okay I'm moving the link back to the top just under the intro - that way, they learn but have a chance to escape to the tutorial before they get too overwhelmed by text :-)

laurapacilio commented on content/source/docs/cloud/run/cli.html.md:
````diff
@@ -41,63 +41,50 @@ To supplement these remote operations, you can also use the optional [Terraform
-rerun this command to reinitialize your working directory. If you forget, other
-commands will detect it and remind you to do so if necessary.
-```
+~> **Note**: The `cloud` block is available in Terraform v1.1 and later and Terraform Enterprise v202201 and later. Previous versions can use the [`remote` backend](/docs/language/settings/backends/remote.html) to configure the CLI workflow and migrate state.
````
@im2nguyen This is something you've called out in my PRs before (the opposite - to change -> to ~> because I'm assuming that's what we do on learn). What are your thoughts here? I agree with Judith that the docs do have a convention of doing notes like -> BUT if we're moving toward a unified experience, maybe we should begin mirroring what we do on Learn?   Casy-by-case maybe isn't the right way to do it...but then again we should start somewhere? Anyway, lmk what you think and once we have consensus, I'll update accordingly!

im2nguyen replied:
I think for now, let's go with Judith's suggestion to make everything consistent (in docs, notes are `->`)

when we have bandwidth, we can create a new PR that replaces all `-> **Note:**` to `~> **Note:**

remember to also make the colon bold!

im2nguyen replied:
I think this note works better right after the first bullet since note is specifically targeted at the `cloud` block

![image](https://user-images.githubusercontent.com/7556325/144982103-072c031a-6581-42e7-a33e-e0fde7617891.png)

laurapacilio replied:
this is an excellent suggestion - i'm going to move it up - thank you!!!

laurapacilio replied:
FYI y'all - I fixed all the notes I could see :-) Thank you for the guidance and for calling that out!

laurapacilio commented on content/source/docs/cloud/run/cli.html.md:
```diff
@@ -41,63 +41,50 @@ To supplement these remote operations, you can also use the optional [Terraform
+      cloud {
+        organization = "my-org"
+        workspaces {
+          tags = ["networking"]
```
It feels like from what I've heard that we a) expect users to be using tags a lot, especially moving forward and b) that we're building more and more functionality that revolves around tags in the future. BUT I defer to others opinions, just wanted to plug tags quickly :)

laurapacilio replied:
jk I'm gonna push up a change that uses "name" here :-)

laurapacilio replied:
Okay Update from my conversation with @chrisarcand - Chris would actually prefer that we use tags here. The reason why is that we think this is now going to be the preferred way to use this feature AND tags are becoming more and more of a big deal in the TFC world. so i'm going to leave it BUT I will add in a sentence of explanation.

im2nguyen commented on content/source/docs/cloud/overview.html.md:
```diff
@@ -44,7 +43,7 @@ Remote execution helps provide consistency and visibility for critical provision
 
 [execution_mode]: ./workspaces/settings.html#execution-mode
```
this is a nit and out of scope, but can we replace the bullets with a numbered list? we introduce it by saying there are "three workflows"

![image](https://user-images.githubusercontent.com/7556325/144981915-aceefb18-b26c-4b89-a164-828db6f0ba6c.png)

laurapacilio replied:
hmmmm I see what you're saying, but I think we want to reserve numbered lists for either steps that must be done in order OR for things that have a clear precedence. I'm going to keep this as bulleted because here all three workflows are valid.

im2nguyen commented on content/source/docs/cloud/run/cli.html.md:
```diff
@@ -111,78 +98,31 @@ If you use a combined repository and [specify a working directory on workspaces]
 CLI-driven runs upload an archive of your configuration directory
 to Terraform Cloud. If the directory contains files you want to exclude from upload,
-you can do so by defining a [`.terraformignore` file in your configuration directory][remote].
+you can do so by defining a [`.terraformignore` file in your configuration directory](/docs/cli/using-terraform-cloud/initialization.html).
```
This file doesn't exist and doesn't exist in the other [PR either](https://github.com/hashicorp/terraform/pull/29947)

im2nguyen replied:
i think we can remove this link if the link doesn't exist tbh

laurapacilio replied:
I have fixed this!

im2nguyen reviewed (approved):
Great job! This looks good 😄 

I think we should update references to `docs/cli/configuring-terraform-cloud/index.html` to `docs/cli/cloud/index.html`. Most of the PRs touch on this 

I tried looking for `docs/cli/configuring-terraform-cloud/index.html` but it wouldn't resolve and I couldn't find it in the [other PR](https://github.com/hashicorp/terraform/pull/29947/files).

laurapacilio commented on content/source/docs/cloud/run/cli.html.md:
````diff
@@ -41,63 +41,50 @@ To supplement these remote operations, you can also use the optional [Terraform
+    ```
 
-Initializing provider plugins...
+1. Run `terraform init`.
````
done! I'll work on adding them back in and push that up :)

laurapacilio commented on content/source/docs/cloud/workspaces/index.html.md:
```diff
@@ -36,18 +29,27 @@ Credentials and secrets | In shell environment or entered at prompts | In worksp
-For more information, see [Terraform Runs and Remote Operations](../run/index.html).
+Both Terraform Cloud and Terraform CLI have features called workspaces, but they function differently.
+
+- Terraform CLI workspaces isolate multiple state files in the same working directory, letting you manage multiple groups of resources with a single configuration. The Terraform CLI does not require you to create CLI workspaces. Refer to [Workspaces](/docs/language/state/workspaces.html) in the Terraform Language documentation for more details.
+
+- Terraform Cloud workspaces are required. They manage collections of infrastructure and are a major component of role-based access in Terraform Cloud. You can grant individual users and user groups permissions for one or more workspaces that dictate whether they can manage variables, perform runs, etc. You cannot manage resources in Terraform Cloud without creating at least one workspace.
```
that's a good point - thank you!!

laurapacilio commented on content/source/docs/cloud/migrate/index.html.md:
````diff
@@ -1,124 +1,43 @@
 
-``` hcl
+```
 terraform {
````
Thank you! I talked to Chris and we added a line of explanation :-)

laurapacilio commented:
> Great Job! Just a few tiny nits but I don't have anything big. Going to pre-approve 🎉
> 
> * The listed version of enterprise won't be out for a while. Are we comfortable making future-facing statements in the docs about when the cloud block will be available?
- FYI - spoke to @chrisarcand about this. In general we agree this is iffy but Chris confirmed that this is a pretty safe statement to make in this ONE case. So I'm leaving it but we can revisit if we later decide otherwise. 
> * In general I'd prefer keeping the code blocks and updating the output. It's nice to confirm that things are working like users would expect, and can also help with our SEO if people google a particular output.
- Good point - i'll update and add back!
> * Version notes have inconsistent formatting across pages
- I believe I have fixed! I changed all the ~ to - .  Thank you!

chrisarcand commented on content/source/docs/cloud/run/index.html.md:
```diff
@@ -61,7 +61,7 @@ Terraform Cloud has three main workflows for managing runs, and your chosen work
 - The [CLI-driven run workflow](./cli.html), which uses Terraform's standard CLI tools to execute runs in Terraform Cloud.
 
-In more abstract terms, runs can be initiated by VCS webhooks, the manual "Start new run" action in the workspace actions menu, the standard `terraform apply` command (with the remote backend configured), and [the Runs API](../api/run.html) (or any tool that uses that API).
+You can initiate Terraform Cloud runs through the manual **Start new run** action in the workspace actions menu, VCS webhooks, the standard `terraform apply` command (with the CLI integration configured), and [the Runs API](../api/run.html) (or any tool that uses that API).
```
This is "Start new plan" now

![image](https://user-images.githubusercontent.com/2430490/145120174-9ea183af-865c-413d-a4f9-81dbd09f8804.png)

laurapacilio replied:
thank you so much for catching this!! I'm pushing up a fix.

chrisarcand commented on content/source/docs/cloud/workspaces/index.html.md:
```diff
@@ -75,3 +77,8 @@ We recommend that organizations break down large monolithic Terraform configurat
+
+## Creating Workspaces
+
+You can create workspaces through the [Terraform Cloud UI](/docs/cloud/workspaces/creating.html) or the [Workspaces API](../api/workspaces.html).
```
Or the CLI, using the Cloud integration! With `tags` configured, you can use `terraform workspace new` and `terraform workspace delete`, etc.

laurapacilio replied:
yes you're right!! let me add :-)

chrisarcand commented on content/source/docs/cloud/workspaces/settings.html.md:
```diff
@@ -48,13 +48,11 @@ The display name of the workspace.
 The default value is "Remote", which instructs Terraform Cloud to perform Terraform runs on its own disposable virtual machines. This provides a consistent and reliable run environment, and enables advanced features like Sentinel policy enforcement, cost estimation, notifications, version control integration, and more.
 
-To disable remote execution for a workspace, change its execution mode to "Local". The workspace will store state, which Terraform can access using the [remote backend][].
+To disable remote execution for a workspace, change its execution mode to "Local". The workspace will store state, which Terraform can access using the [CLI integration](/docs/cli/cloud/index.html).
```
(This is in critique of the original wording, so for the purposes of this PR, I understand if you want to ignore it)

This sentence is too vague, I think. As someone who doesn't yet know anything about the various workflows, I don't know what "Terraform can access using the CLI integration" means. 

Something like...(this is worded poorly, hastily, but gets at what I mean)

> To disable remote execution for a workspace, change its execution mode to "Local". This mode is used solely for the CLI-driven run workflow [link]. In this mode, the workspace will be used to store state and variables for using Terraform on a local command line, with the process running on your local machine.

laurapacilio replied:
Fair - thank you!! I updated :-)
